sdk/cmake/set_wine_module.cmake

```diff
@@ -0,0 +1,7 @@
+
+function(set_wine_module TARGET)
+    target_include_directories(${TARGET} SYSTEM BEFORE PRIVATE
+        ${REACTOS_SOURCE_DIR}/sdk/include/reactos/wine
+        ${REACTOS_BINARY_DIR}/sdk/include/reactos/wine)
```

Comment:
`${REACTOS_BINARY_DIR}/...` looks unneeded (yet), isn't it?

Reply:
`sdk/include/reactos/wine/CMakeLists.txt` assumes `${REACTOS_BINARY_DIR}/sdk/include/reactos/wine` files generation.

Reply:
Yeah, but none of the dlls included it before, so why is it needed now?
It looks like only `base/services/rpcss/`, which is not touch (yet) by this PR, uses it.
https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=%24%7BREACTOS_BINARY_DIR%7D%2Fsdk%2Finclude%2Freactos%2Fwine